selfservice/flow/verification/handler_test.go

```diff
@@ -84,6 +84,40 @@ func TestGetFlow(t *testing.T) {
 		})
 	})
 
+	t.Run("case=valid with return_to", func(t *testing.T) {
+		t.Run("type=browser", func(t *testing.T) {
+			conf.MustSet(config.ViperKeyURLsAllowedReturnToDomains, []string{"https://www.ory.sh/"})
```

Comment:
```suggestion
	t.Run("case=valid with return_to", func(t *testing.T) {
		conf.MustSet(config.ViperKeyURLsAllowedReturnToDomains, []string{"https://www.ory.sh/"})
		
		t.Run("type=browser", func(t *testing.T) {
```
and then you can remove it from all the individual tests.